Installing-and-Configuring/OS/Generic.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Generic
+permalink: /OS/Generic/
+parent: Operating Systems
+grand_parent: Installing and configuring
+nav_order: 1
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Generic OS Installation
+===
+
+Insert OS installation media into one of the USB3 ports (blue on Thinkpads).
+ Turn the system on and navigate to the USB boot in the graphical heads menu. 
```

Comment:
Insert OS installation media preferably into one of the USB3 ports for speedier installation (blue ports ).

Reply:
incorporated with a few word changes.  will push in a few minutes.